modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -0,0 +1,635 @@
+|Removed without replacement.
+
+|===
+====
+
+
+==== Changes to default values of configuration settings
+
+These changes affect only new installations or upgrades that involve replacing existing configuration files.
+
+.See changes to default values in Neo4j 2025.01.
+[%collapsible]
+====
+[options=header, cols="2m,1,1"]
+|===
+| Name
+| Old default value
+| New default value
+
+|xref:configuration/configuration-settings.adoc#config_dbms.cypher.infer_schema_parts[dbms.cypher.infer_schema_parts]
+|`OFF`
+|`MOST_SELECTIVE_LABEL`
+
+|xref:configuration/configuration-settings.adoc#config_db.logs.query.annotation_data_format[db.logs.query.annotation_data_format] label:dynamic[]
+|`CYPHER`
+|`JSON`
+
+|xref:configuration/configuration-settings.adoc#config_dbms.databases.seed_from_uri_providers[dbms.databases.seed_from_uri_providers] label:enterprise[Enterprise Edition]
+|`S3SeedProvider,CloudSeedProvider`
+|`CloudSeedProvider`
+
+|xref:configuration/configuration-settings.adoc#config_server.metrics.csv.rotation.compression[server.metrics.csv.rotation.compression] label:enterprise[Enterprise Edition]
+|`NONE`
+|`ZIP`
+
+|xref:configuration/configuration-settings.adoc#config_server.panic.shutdown_on_panic[server.panic.shutdown_on_panic] label:enterprise[Enterprise Edition]
+|`false`
+|`true`
+
+|xref:configuration/configuration-settings.adoc#config_server.logs.config[server.logs.config]
+|`conf/server-logs.xml`
+|`server-logs.xml`
+
+|xref:configuration/configuration-settings.adoc#config_server.logs.user.config[server.logs.user.config]
+|`conf/user-logs.xml`
+|`user-logs.xml`
+|===
+====
+
+
+==== Changes to valid values of configuration settings
+
+The `server.logs.config` and the `server.logs.user.config` are changed to depend on the provided path for the _conf_ directory instead of having the _conf_ directory encoded in the default values.
+
+.Updated configuration settings
+[options=header, cols="2m,3,3"]
+|===
+| Name
+| Old valid value
+| New valid value
+
+|`server.logs.config`
+|A path. If relative, it is resolved from `server.directories.neo4j_home`.
+|A path. If relative, it is resolved from `server.directories.configuration`.
+
+|`server.logs.user.config`
+|A path. If relative, it is resolved from `server.directories.neo4j_home`.
+|A path. If relative, it is resolved from `server.directories.configuration`.
+|===
+
+
+
+[role=label--enterprise]
+=== Neo4j metrics
+
+[role=label--enterprise]
+==== Removed metrics
+
+.See removed metrics in Neo4j 2025.01.
+[%collapsible]
+====
+[options="header", cols="1,1"]
+|===
+|Name|Description
+
+2+|**link:{neo4j-docs-base-uri}/operations-manual/5/monitoring/metrics/reference/#raft-core-metrics[Raft core metrics] - replaced accordingly by the link:{neo4j-docs-base-uri}/operations-manual/5/monitoring/metrics/reference/#raft-metrics[Raft metrics]**
```

Comment:
We don't need descriptions on this page. I think having a simple table with the old metric(link) and the replacement (link) would make it easier to grasp the change.

Reply:
We can improve it later. BTW, metrics don't have their own links. They're grouped in tables. I tried different scenarios here.  We can leave here only metrics classes, but that is not very informative

Reply:
Sure. Sounds like a project 😄